Artifacts/MsSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,45 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @claim_Name NVARCHAR(MAX) = 'http://ed-fi.org/ods/identity/claims/domains/educationOrganizations';
+DECLARE @educationOrganizationsResourceClaimId INT;
+DECLARE @claimSet_Id INT;
+DECLARE @claimSet_Name NVARCHAR(MAX) = 'Bootstrap Descriptors and EdOrgs';
+DECLARE @msg NVARCHAR(MAX);
+IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = @claim_Name)    
+BEGIN
+	SET @msg = 'ClaimName does not exist';
```

Comment:
Replace <kbd>TAB</kbd>s with spaces.

Reply:
This message is less expressive than the Postgres version which include the claim name in the message. Modify this message to match the Postgres one.

Reply:
yes , I formatted with indent and also modified message match with Postgres